### DISCUSSION TITLE
Cannot submit entry for GitHub Action Hackathon

### DISCUSSION THREAD

**Discussion by giorgosneokleous93** [community/community#25181] [Actions]
<p>I’m unable to submit an entry to the GitHub action Hackathon (<a href="https://githubhackathon.com/" rel="nofollow noopener">https://githubhackathon.com/</a>)</p>
<p>When I click on the repo for submission I get a warning icon with a tooltip text “null”.</p>
<p>Does anyone knows what’s going on?</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p><a class="mention" href="/giorgosneokleous93">@giorgosneokleous93</a> I saw you have post a comment in the Featured Event: GitHub Actions Hackathon ticket <a href="https://github.community/t/featured-event-github-actions-hackathon/13091/42">https://github.community/t5/Events/Featured-Event-GitHub-Actions-Hackathon/m-p/51362/highlight/true#M268</a></p>
<p>That’s the more appropriate place where you could contact to Github Action Hackathon team directly. So please pay attention to that ticket. Thank you for your understanding.</p>
